README.md

```diff
@@ -493,7 +493,7 @@ are listed.
 
 ### 80 (experimental)
 
-Available since 1.0.7 (XXX).
+Available since 1.0.7 (2020-07-11).
```

Comment:
Hm, wasn't this changed to 1.0.10 in the other PR?

Reply:
Fixed.